proto/regen/ecocredit/v1/tx.proto

```diff
@@ -35,11 +35,20 @@ service Msg {
   rpc MintBatchCredits(MsgMintBatchCredits)
       returns (MsgMintBatchCreditsResponse);
 
-  // MsgSealBatch sets the `batch.open` attribute to false. Sealed batch
+  // SealBatch sets the `batch.open` attribute to false. Sealed batch
   // can't issue more credits. Once batch is sealed it can't be toggled any
   // more. Only batch creator can seal a batch.
   rpc SealBatch(MsgSealBatch) returns (MsgSealBatchResponse);
 
+  // MintPendingBatchCredits creates a pending batch issuance of credits for an issuer to approve
+  // within a specified amount of time.
+  rpc MintPendingBatchCredits(MsgMintPendingBatchCredits) returns (MsgMintPendingBatchCreditsResponse);
```

Comment:
if this creates a new batch, then maybe we should rename to `CreatePendingBatch`?

Reply:
It might create a new batch, but not necessarily (e.g. if a user sets `batch_denom` instead of a `PendingBatchInfo`.  Alternatively, I was thinking about splitting this out into separate RPC calls:
- one for creating a new batch (and optionally adding issuances to it)
- one for minting new credits to an existing batch

Do you think that API is cleaner?

Reply:
yes, in this case 2 RPC is better.